test/fields/collections/Row/index.ts

```diff
@@ -60,6 +89,9 @@ const RowFields: CollectionConfig = {
               type: 'text',
             },
           ],
+          admin: {
+            width: '30%',
+          },
         },
         {
           label: 'Collapsible within a row',
```

Comment:
Nit. Perhaps the label could reflect that the first one is 30% to differentiate it from the other.

Reply:
Good idea, added here: https://github.com/payloadcms/payload/pull/7940/commits/ef6fe9ca9a55b54d1d634b3421f9fd80881ae0d8#diff-a7c8a52eb3baea0a3afdd63367fccb9a2c09512de56e29b1c61c083a4c3fd932R83